extension/content.js

```diff
@@ -154,6 +162,16 @@ function addReleasesTab() {
 
 	if (!hasReleases) {
 		$repoNav.append($releasesTab);
+
+		if (isRepoRoot()) {
+			const releasesCount = $('.numbers-summary a[href$="/releases"] .num').text().trim();
+			appendReleasesCount(releasesCount);
+			chrome.storage.local.set({[releasesCountCacheKey]: releasesCount});
+		} else {
+			chrome.storage.local.get(releasesCountCacheKey, items => {
+				appendReleasesCount(items[releasesCountCacheKey]);
+			});
+		}
```

Comment:
Maybe put all this in a function and call it from here for clarity. `cacheReleasesCount()`.